playground/src/PlaygroundView.tsx

```diff
@@ -69,14 +71,16 @@ export class PlaygroundView extends React.Component<IPlaygroundViewProps, IPlayg
 
   private _renderInputBox(): React.ReactNode {
     return (
-      <FlexColDiv className='playground-input-box' style={ { flex: 1 } }>
+      <FlexColDiv className='playground-input-box' style={ { flex: 1, width: '50%' } }>
```

Comment:
>width: '50%' [](start = 70, length = 12)

Why do we need width=50%?

Reply:
Otherwise monaco won't detect that its container has changed size when the window resizes.

---
In reply to: [223514879](https://github.com/Microsoft/tsdoc/pull/90#discussion_r223514879) [](ancestors = 223514879)

Reply:
Removed.

---
In reply to: [223518767](https://github.com/Microsoft/tsdoc/pull/90#discussion_r223518767) [](ancestors = 223518767,223514879)